sdk/jdbc/azure-identity-providers-jdbc/src/main/java/com/azure/identity/providers/jdbc/enums/AuthProperty.java

```diff
@@ -0,0 +1,195 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.identity.providers.jdbc.enums;
+
+
+import com.azure.identity.AzureAuthorityHosts;
+
+import java.util.Map;
+import java.util.Properties;
+
+/**
+ * Contains authentication property used to resolve token credential.
+ */
+public enum AuthProperty {
```

Comment:
this should not be a public API
As we discussed, all the classes except the two auth plugins should be implementation